app/components/primer/alpha/action-menu-element.ts

```diff
@@ -22,9 +23,43 @@ class ActionMenuElement extends HTMLElement {
     // eslint-disable-next-line no-invalid-this
     this.#menuEl && this.#menuEl.querySelectorAll('[role="menuitem"],[role="menuitemcheckbox"],[role="menuitemradio"]')
 
+  get anchorAlign(): AnchorAlignment {
+    return (this.getAttribute('data-anchor-align') || 'start') as AnchorAlignment
+  }
+
+  set anchorAlign(value: AnchorAlignment) {
+    this.setAttribute('data-anchor-align', value)
+  }
+
+  get anchorSide(): AnchorSide {
+    return (this.getAttribute('data-anchor-side') || 'outside-bottom') as AnchorSide
+  }
+
+  set anchorSide(value: AnchorSide) {
+    this.setAttribute('data-anchor-align', value)
+  }
+
   connectedCallback() {
     if (!this.#buttonEl) return
 
+    // THIS IS TEMPORARY AND SHOULD BE REPLACED BY PRIMER CSS CLASS
+    const style = document.createElement('style')
+    style.innerHTML = `
+        action-menu {
+          position: relative;
+        }
+        action-menu ul {
+          position: absolute;
+          width: 160px;
+          background-color: var(--color-canvas-overlay);
+          z-index: 1000000;
+          border: $border-width $border-style var(--color-border-default);
+          border-radius: $border-radius;
+          box-shadow: var(--color-shadow-large);
+        }
+    `
+    document.querySelector('body')?.appendChild(style)
```

Comment:
@langermank, This is temporary CSS I added particularly for styling the `ul[role='menu']` to validate the positioning logic works. I'm assuming that there will be a Primer CSS we want here?

(Note: I originally had styles defined in Shadow DOM but had the worst time figuring out how to style nested elements after markup change. This is temporary anyways!)